src/modules/poll/index.js

```diff
@@ -1,29 +1,41 @@
 
 const command = require('../command');
-const { discord: { broadcastChannelName } } = require('../../config');
+// const { discord: { broadcastChannelName } } = require('../../config');
 
+/**
+ * reusable function to form an output string, containing the info about a single poll
+ */
 const getPollStringInfo = function (i18n, {
     dateCreated,
     question,
     options,
     pollId,
+    isOpen,
 }) {
+    let i18nOption = 'poll.header';
+    if (isOpen === false) i18nOption += '.wstatus';
```

Comment:
its very hard to understand... please dont concate CONSTANTS (i18n selector is constants)

Reply:
and prefer to line break and use curly braces in single if statment (for better readable)